fe/fe-core/src/main/java/com/starrocks/connector/jdbc/JDBCMetadata.java

```diff
@@ -67,22 +71,16 @@ public JDBCMetadata(Map<String, String> properties, String catalogName) {
     public JDBCMetadata(Map<String, String> properties, String catalogName, HikariDataSource dataSource) {
         this.properties = properties;
         this.catalogName = catalogName;
-        try {
-            String driverName = getDriverName();
-            Class.forName(driverName);
-        } catch (ClassNotFoundException e) {
-            LOG.warn(e.getMessage());
-            throw new StarRocksConnectorException("doesn't find class: " + e.getMessage());
-        }
         if (properties.get(JDBCResource.DRIVER_CLASS).toLowerCase().contains("mysql")) {
             schemaResolver = new MysqlSchemaResolver();
         } else if (properties.get(JDBCResource.DRIVER_CLASS).toLowerCase().contains("postgresql")) {
             schemaResolver = new PostgresSchemaResolver();
         } else if (properties.get(JDBCResource.DRIVER_CLASS).toLowerCase().contains("mariadb")) {
             schemaResolver = new MysqlSchemaResolver();
+        } else if (properties.get(JDBCResource.DRIVER_CLASS).toLowerCase().contains("oracle")) {
+            schemaResolver = new OracleSchemaResolver();
         } else {
-            LOG.warn("{} not support yet", properties.get(JDBCResource.DRIVER_CLASS));
-            throw new StarRocksConnectorException(properties.get(JDBCResource.DRIVER_CLASS) + " not support yet");
+            schemaResolver = new DefaultJDBCSchemaResolver();
```

Comment:
For catalog types that cannot be supported, it is better to explicitly throw an error rather than providing a default value.